### PR TITLE
Bulk ingest videos with corresponding vtt caption files

### PR DESCRIPTION
**Pull request by hackartisan**
- Include "videos" in the bulk ingest heading"
- Accommodate bulk ingest of video with vtt
- Make ingest folder job accept mp4

advances #6180

### CODE REVIEW

File: spec/views/bulk_ingest/show.html.erb_spec.rb [hunk 1]
```diff
@@ -12,4 +12,33 @@
...
+        assign :resource_class, ScannedResource
+        assign :visibility, []
+        assign :states, []
+        assign :collections, []
+        render
+
+        # rendered is full of awkward line breaks and large spaces
+        xml = Nokogiri.parse(rendered)
+        heading_text = xml.css("h2").text.split(" ").join(" ")
+        expect(heading_text).to eq "Bulk Ingest Scanned Resources, Videos, or Vendor Bags"
```
Comment (tpendragon) [inline spec/views/bulk_ingest/show.html.erb_spec.rb:28]: I think in view specs we have Capybara helpers. You should be able to do `expect(rendered).to have_css("h2", text: "Bulk Ingest Scanned Resources, Videos, or Vendor Bags"`

**Review by tpendragon — CHANGES_REQUESTED** [review]
Small change to try.

File: spec/views/bulk_ingest/show.html.erb_spec.rb (hunk 1 above)
Comment (hackartisan) [reply spec/views/bulk_ingest/show.html.erb_spec.rb:28]: ha, yay!

**Review by tpendragon — APPROVED** [review]

**Comment by tpendragon** [comment]
Woo, good work!

**Pull request merged**